app/views/active_storage/blobs/_blob.html.erb

```diff
@@ -1,14 +1,35 @@
-<figure class="attachment attachment--<%= blob.representable? ? "preview" : "file" %> attachment--<%= blob.filename.extension %>">
```

Comment:
We definitely need to keep this outer `figure` element for general backwards-compatibility friendliness to downstream developers who are trying to upgrade existing apps without springing unexpected changes on their users. (For example, people may have written custom CSS targeting these `figure` elements.)